### PR TITLE
Add Check Personal Information Page

### PR DESCRIPTION
**Pull request by David-Feetenby**
This adds a new page which assessors will use to check the personal of the application. For now it just displays the personal information and a button which takes the user back to the overview page.

### CODE REVIEW

**Comment by David-Feetenby** [comment]
[Check personal information](https://trello.com/c/9yQ9Iakd/836-check-personal-information)

**Review by thomasleese — COMMENTED** [review]
Change looks good, just left a couple of comments.

File: lib/tasks/example_data.rake [hunk 1]
```diff
@@ -5,10 +5,10 @@ namespace :example_data do
...
     end
 
-    if Teacher.any?
-      puts "Noop as DB already contains data"
-      exit
-    end
+    # if Teacher.any?
+    #   puts "Noop as DB already contains data"
+    #   exit
+    # end
```
[the remark this answers is not in the document]
Comment (David-Feetenby) [reply lib/tasks/example_data.rake:11]: Oops cheers

File: app/views/assessor_interface/check_personal_informations/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<% content_for :page_title, "Check personal information" %>
+    <% content_for :back_link_url, assessor_interface_application_form_path(@application_form) %>
+
+        <h1 class="govuk-heading-xl">Check personal information</h1>
+
+        <%= render "shared/application_form/personal_information_summary" , application_form: @application_form,
+            personal_information: @personal_information, changeable: false %>
+
+            <%= govuk_button_link_to "Continue" , [:assessor_interface, @application_form] %>
```
[the remark this answers is not in the document]
Comment (David-Feetenby) [reply app/views/assessor_interface/check_personal_informations/show.html.erb]: That better?

File: app/views/assessor_interface/check_personal_informations/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<% content_for :page_title, "Check personal information" %>
+
+<% content_for :back_link_url, assessor_interface_application_form_path(@application_form) %>
+
+<h1 class="govuk-heading-xl">Check personal information</h1>
+
+<%= render "shared/application_form/personal_information_summary" , application_form: @application_form,
+   personal_information: @personal_information, changeable: false %>
```
Comment (thomasleese) [inline app/views/assessor_interface/check_personal_informations/show.html.erb:8]:
```suggestion
<%= render "shared/application_form/personal_information_summary",
           application_form: @application_form,
           changeable: false %>
```

I'd suggest having the indentation like this (we also don't need to pass a `personal_information` variable).

File: app/views/assessor_interface/check_personal_informations/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+   
+<%= govuk_button_link_to "Continue" , [:assessor_interface, @application_form] %>
```
Comment (thomasleese) [inline app/views/assessor_interface/check_personal_informations/show.html.erb:10]:
```suggestion
<%= govuk_button_link_to "Continue", [:assessor_interface, @application_form] %>
```

**Review by thomasleese — APPROVED** [review]

**Pull request merged**